package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-api",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
it's not a breaking change since this method returned string in previous versions.
when it stopped to return a string, a bug happened.
CLI takes the latest version now.